Jenkinsfile.daily

```diff
@@ -0,0 +1,65 @@
+properties([
+    pipelineTriggers([cron('H H(1-3) * * *')]),
```

Comment:
1h sometimes people are working. What about 2-6? or even 3-6?

Reply:
The impact here is minimal for now with under a minute of execution time. It can be moved around when needed later.